src/xcode/ENA/ENA/Source/Services/CCLService/__tests__/CCLConfiguationTests.swift

```diff
@@ -171,7 +188,16 @@ class CCLServiceConfigurationTests: CCLServiceBaseTests {
 		)
 		let cache = try cache(with: Locator.CCLConfiguration(isFake: false), eTag: eTag, date: yesterday, responseData: oldCCLConfigurationData)
 		let restServiceProvider = RestServiceProvider(session: stack.urlSession, cache: cache)
-		let cclService = CCLService(restServiceProvider, appConfiguration: appConfiguration, cclServiceMode: [.configuration], signatureVerifier: MockVerifier())
+		let resource = MockCCLConfigurationResource(defaultModel: nil)
+
+		let cclService = CCLService(
+			restServiceProvider,
+			appConfiguration: appConfiguration,
+			cclServiceMode: [.configuration],
+			signatureVerifier: MockVerifier(),
+			cclConfigurationResource: resource
```

Comment:
I think it would be good if we would have a component which loads the default. This component could be used in the resource and also be injected here.
This way we would not touch the service implementation. And avoid the cross dependency between the CCLService (CCL processing) and the CCLConfigurationResource (HTTP).